examples/flax/language-modeling/run_clm_flax.py

```diff
@@ -574,9 +574,9 @@ def tokenize_function(examples):
         if block_size > config.max_position_embeddings:
             logger.warning(
                 f"The tokenizer picked seems to have a very large `model_max_length` ({tokenizer.model_max_length}). "
-                "Picking 1024 instead. You can change that default value by passing --block_size xxx."
+                f"Using block_size={config.max_position_embeddings} instead. You can change that default value by passing --block_size xxx."
             )
-            block_size = 1024
```

Comment:
Hey @pphuc25, thanks for opening this PR! I believe a block size of 1024 was chosen since it's a multiple of 128. Multiples of 128 tend to work extremely well on TPU - these magic multiples give much faster training efficiency. 

Perhaps we can chose the minimum of `(1024, config.max_position_embeddings)` for the block size in this case? Note that the user is warned that the block size is defaulting to 1024, so they are free to pass the `--block_size` argument to the training script if they feel this is necessary to improve performance.